static/css/partials/_header.scss

```diff
@@ -60,12 +57,6 @@
     position: relative;
   }
 
-  a.nav-button {
-    &:focus {
-      outline: 1px dotted $black;
-    }
-  }
-
```

Comment:
nit: Removing this shows a blue highlight when settings and myshots button are pressed in chrome